test/cli/unit/test_teos_cli.py

```diff
@@ -70,6 +71,29 @@ def test_register():
     )
 
 
+@responses.activate
```

Comment:
Well, I wanted to test that indeed there is no network request. In principle, the function could do a network requests first, then validate the user id and throw an error because it's invalid; that would be semantically correct, but still result in an unnecessary network request.
Checking that there are no network requests forces to do validation _before_ the network request in order to pass the test (similarly for other tests below).

Reply:
Seems overkill. What would you provide a user_id to a service and then check if what you provided was correct before getting the response?

Reply:
Removed in 8bb4003e1446e68ec4f253a9f715102b83ff6765.